app/assets/stylesheets/pin-show.css.scss

```diff
@@ -0,0 +1,117 @@
+$primary-background-color: rgba(237, 242, 242, 0.6);
+
+
+.container-fluid{
```

Comment:
I don't really recommend messing up with Bootstrap's own classes, but rather trying to add another class that takes you bg color?